### ISSUE TITLE
release 2.3.1 breaks if concurrency is not set in opts

### ISSUE THREAD

**Issue by elfosardo** [mtreinish/stestr#235]
When running stestr without the opts concurrency, the default for concurrency is 0.
In the code I can read:
```
:param int concurrency: "How many processes to use. The default (0)
        autodetects your CPU count and uses that.
```

Unfortunately, when the concurrency is not set in opts, an error will be raised:
```
Traceback (most recent call last):
  File "/home/rpittau/git_openstack/ironic-specs/.tox/linters/bin/ostestr", line 10, in <module>
    sys.exit(main())
  File "/home/rpittau/git_openstack/ironic-specs/.tox/linters/lib/python3.7/site-packages/os_testr/ostestr.py", line 265, in main
    exit(ostestr(sys.argv[1:]))
  File "/home/rpittau/git_openstack/ironic-specs/.tox/linters/lib/python3.7/site-packages/os_testr/ostestr.py", line 261, in ostestr
    return _select_and_call_runner(opts, regex, others)
  File "/home/rpittau/git_openstack/ironic-specs/.tox/linters/lib/python3.7/site-packages/os_testr/ostestr.py", line 214, in _select_and_call_runner
    black_regex=opts.black_regex)
  File "/home/rpittau/git_openstack/ironic-specs/.tox/linters/lib/python3.7/site-packages/os_testr/ostestr.py", line 168, in call_testr
    black_regex=black_regex)
  File "/home/rpittau/git_openstack/ironic-specs/.tox/linters/lib/python3.7/site-packages/stestr/commands/run.py", line 335, in run_command
    concurrency = _to_int(concurrency)
  File "/home/rpittau/git_openstack/ironic-specs/.tox/linters/lib/python3.7/site-packages/stestr/commands/run.py", line 39, in _to_int
    i = int(possible)
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

This issue can be fixed with the patch in commit:
https://github.com/mtreinish/stestr/commit/ac8dac07f5faa487254c649e7411515616392403

**Comment by mtreinish** [comment]
This issue is already fixed by #233 as you referenced in the comment. We don't need to have an open issue for a fixed issue. That all being said this is more of a bug in ostestr because it is passing explicitly passing None to stestr for concurrency, not simply neglecting to pass anything for the field. This isn't what stestr expects, as you point out, the docs say it's an int, not None.

**Comment by elfosardo** [comment]
@mtreinish thank you for your comment.
That's true, the issue is fixed in master, but my point is that it's not fixed for release 2.3.1, which is the latest. Actually, the issue was introduced in that version because of the introduction of the _to_int() function, that was fixing another issue #231 
It's a shame that the fix didn't make into release 2.3.1, would be great if we could have a new release to include the fix.
Thank  you!
